docs/src/modules/components/AppSearch.js

```diff
@@ -45,12 +45,16 @@ function initDocsearch(userLanguage) {
     }
 
     initialized = docsearchInput;
-    window.docsearch({
+    const search = window.docsearch({
       apiKey: '1d8534f83b9b0cfea8f16498d19fbcab',
       indexName: 'material-ui',
       inputSelector: '#docsearch-input',
       algoliaOptions: {
         facetFilters: ['version:master', `language:${userLanguage}`],
+        hitsPerPage: 40,
```

Comment:
Random thought: I'm wondering if it's not too many options 🤔?

Reply:
Why would it be too many?

Reply:
The initial thought comes from the tradeoff Algolia took in the default value of this option. 
We can take the query "modal", let's say we are looking for the API, with 40 results, we have to scroll 10 results, with the default value, we can see it without scrolling. 

https://www.smartinsights.com/search-engine-optimisation-seo/seo-analytics/comparison-of-google-clickthrough-rates-by-position/ could be an interesting resource. Seems that most people stop looking after 10 results.

Reply:
> We can take the query "modal", let's say we are looking for the API, with 40 results, we have to scroll 10 results, with the default value, we can see it without scrolling.

I don't undestand this point. How can something be hidden with less results?

Reply:
> Seems that most people stop looking after 10 results.

So lets improve the situation for the people who keep looking.

Reply:
Agree, if we can help the few without harming the many, it's good to take 👍